python/lsst/meas/modelfit/display/interactive.py

```diff
@@ -25,7 +25,6 @@
 import collections
 import numpy
 import copy
-from matplotlib import pyplot
 import matplotlib.colors
 import mpl_toolkits.mplot3d
```

Comment:
Actually, it looks like `import matplotlib.colors` should go in `optimizerDisplay.py` as well.